File: MediaBrowserTv/app/src/main/java/tv/mediabrowser/mediabrowsertv/startup/SelectServerActivity.java
package tv.mediabrowser.mediabrowsertv.startup;

import android.app.Activity;
import android.os.Bundle;

import tv.mediabrowser.mediabrowsertv.R;

public class SelectServerActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_servers);
    }
}

File: app/src/main/java/org/jellyfin/androidtv/ui/startup/SelectServerActivity.kt
package org.jellyfin.androidtv.ui.startup

import android.os.Bundle
import org.jellyfin.androidtv.ui.shared.BaseActivity

class SelectServerActivity : BaseActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		supportFragmentManager
			.beginTransaction()
			.replace(android.R.id.content, SelectServerFragment())
			.commit()
	}
}
